mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -985,6 +1008,12 @@ class WidgetAdapter(
             }
             return widget.url
         }
+
+        override fun onClick(v: View?) {
+            boundWidget?.let {
+                loadWidget(boundWidget!!)
```

Comment:
it

Reply:
🤦‍♂ Thanks. I found this issue by myself, but you were one minute faster :)